src/android/java/io/jxcore/node/BtToRequestSocket.java

```diff
@@ -40,9 +38,7 @@
 
     private int mHTTPPort = 0;
 
-    final String TAG = "--BtCon-CLIENT-Socket";
-
-    boolean mStopped = false;
+    private boolean mStopped = false;
 
     public BtToRequestSocket(BluetoothSocket socket, BtSocketDisconnectedCallBack handler,ReadyForIncoming callback) {
         print_debug("Creating BtConnectedRequestSocket");
```

Comment:
I don't see the point you are expressing. Each catch is calling Disconnected(), which will clean up the class instance in BtConnectorHelper side. Also right after calling Disconnected there is return; which stops the execution of run.